### PR TITLE
chore: update to use `getaddressinfo` instead of `listunspent`

### PR DESCRIPTION
**Pull request by RafilxTenfen**

### CODE REVIEW

File: docs/create-phase1-staking.md [hunk 1]
```diff
@@ -31,46 +31,55 @@ of the staker public key, which corresponds to the public key
...
-    "confirmations": 30,
-    "spendable": true,
-    "solvable": true,
-    "desc": "wpkh([4b45934d/0h/0h/0h]020721ef511b0faee2a487a346fdb96425d9dd7fa79210adbe7b47f0bcdc7e29de)#kxsmmtvy",
-    "parent_descs": [
-    ],
-    "safe": true
-  }
-]
+  getaddressinfo <addr>
```
Comment (KonradStaniec) [inline docs/create-phase1-staking.md:43]: I would mention somewhere that you create new address (`addr`)in your wallet  by using - https://developer.bitcoin.org/reference/rpc/getnewaddress.html

Comment (KonradStaniec) [reply docs/create-phase1-staking.md:43]: tbh I would add example of `bitcoin-cli getnewaddress` here also, as people like to not click on the links 😅

Comment (RafilxTenfen) [reply docs/create-phase1-staking.md]: https://github.com/babylonchain/btc-staker/pull/147/commits/ab8c6054958d4f7720446c9c3c7d422be74a3174

**Review by KonradStaniec — APPROVED** [review]

File: docs/create-phase1-staking.md [hunk 2]
```diff
@@ -23,6 +23,19 @@ It can be generated in offline mode without any external connections.
 It only requires the specification of a funding Bitcoin public key. Wallet
 creation and generating a Bitcoin public key for it are covered by
 [this guide (steps 2 to 2.4)](../README.md#2-setting-up-a-bitcoin-node).
+In the guide to the last command needed to continue this document is
+the [`bitcoin-cli getnewaddress`](https://chainquery.com/bitcoin-cli/getnewaddress).
```
Comment (vitsalis) [inline docs/create-phase1-staking.md:27]: I don't understand the above sentence

**Review by vitsalis — APPROVED** [review]
lgtm apart from a sentence that I can't decode 😄

File: docs/create-phase1-staking.md (hunk 2 above)
Comment (RafilxTenfen) [reply docs/create-phase1-staking.md]: kkk It kind of got weird to repeat the command from the guide we pointed, tried to make it better here 
https://github.com/babylonchain/btc-staker/pull/147/commits/48776246b7506533f349f702ca7139f719b5a49c

**Pull request merged**